frontend/app/src/locales/en.json

```diff
@@ -1450,7 +1450,12 @@
     "blockchain_balances": {
       "add": "Add blockchain address",
       "title": "blockchain",
-      "tooltip": "Aggregate value of all configured addresses in each blockchain."
+      "tooltip": "Aggregate value of all configured addresses in each blockchain.",
```

Comment:
Do we really need the tooltip? were people confused?

Reply:
the tooltip is there the whole time, but not showed. But yeah I think nobody need it. I will remove it.